src/main/java/com/onwelo/tinvalidator/validator/BEValidator.java

```diff
@@ -0,0 +1,23 @@
+package com.onwelo.tinvalidator.validator;
+
+import java.util.regex.Pattern;
+
+public class BEValidator implements Validator {
+
+    private static final Pattern PATTERN = Pattern.compile("[0-9]{11}");
+
+    public Pattern getPattern(){
+        return PATTERN;
+    }
+
+    public boolean computeControlSum(String tin) {
+        try {
+            assert Integer.parseInt(tin.substring(2,4)) <= 12;
+            assert Integer.parseInt(tin.substring(4,6)) <= 31;
+            int sum = Integer.parseInt(tin.substring(0,9));
+            return 97 - (sum % 97) == Integer.parseInt(tin.substring(9, 11));
```

Comment:
There is second scenario for this algorithm : 
1.Get the remainder of the division by 97 of the number composed by number 2 and C1, C2, C3, C4, C5, C6, C7, C8 and C9;
I don't see anywhere here concating number 2 to tin substring